### ISSUE TITLE
BladeBurner - estimated succes chance doesnt rise

### ISSUE THREAD

**Issue by NoOneNeedsThat** [bitburner-official/bitburner-src#1826]
be in BN 10.2
have all Blackops in BladeBurner finished
(BladeBurner Rank 11.5mill)
![grafik](https://github.com/user-attachments/assets/4be076a9-e191-4a3c-8cdb-070996006991)

have all augments installed (conventional or grafting)
have attributes in high values
![grafik](https://github.com/user-attachments/assets/4229c00a-8f20-42ba-b9fd-da0bcc32343b)

have no sleeves doing BladeBurner tasks

doing field analyses dont give clear estimated success chances, even after running several 100 times
![grafik](https://github.com/user-attachments/assets/ad056ed8-0d89-4a6b-bc9c-3b823da658e0)

even lowering level doesnt give a clear est. success chance
![grafik](https://github.com/user-attachments/assets/3e459e93-f13d-447b-bc9f-a4ece630bd57)

maybe an issue because of bad attribute ratio????? (eg. str vs cha ???)

**Comment by NoOneNeedsThat** [comment]
i think i found out what the problem is:
![grafik](https://github.com/user-attachments/assets/eab666e5-79f0-41e3-ab09-fd2ee4f8efa2)

is that intended????

**Comment by catloversg** [comment]
Check my comment at https://github.com/bitburner-official/bitburner-src/issues/1575#issuecomment-2288729650.

In the next version, we don't have this problem.

**Comment by NoOneNeedsThat** [comment]
thx for clarification.
in fact, the comments in #1575 only apply partially to my situation, as i did tracking and other contracts regularily. 

i told my script to check for success chance before starting any action (as i dont like failures). i also told my script to do a field analysis like every 5th action (which should keep the pop estimates accurate) or if getActionEstimatedSuccessChance[0] of an action is smaller than getActionEstimatedSuccessChance[1]
i was pretty sure to get a proper estimate this way

so, in conclusion to your comments, i either have to switch to a city with est pop or force a tracking on a scheduled basis? (for the mechanics working properly)
is the current BladeBurner city relevant for the tracking contracts or will the tracking contract improve the est pop in all cities?

as a fun fact, #1575 seem to have the issues in the same bitnode as me (BN10.2)

**Comment by catloversg** [comment]
Both pop and est. pop are usually not 0 (except special cases). Running Tracking is only a workaround when est. pop drops to 0. This situation is rare. If it happens, it's very likely that pop is also 0 or very low. It's best to move to another city.

> is the current BladeBurner city relevant for the tracking contracts or will the tracking contract improve the est pop in all cities?

Current city.

**Comment by gmcew** [comment]
Good explanation from Catloversg; I'll close this as it's heavily related to #1575.
